risc0/binfmt/Cargo.toml

```diff
@@ -26,3 +26,4 @@ tracing = { version = "0.1", default-features = false }
 [features]
 default = ["std"]
 std = ["anyhow/std", "elf/std", "tracing/log", "tracing/std", "risc0-zkp/std", "serde/std"]
+recursion = ["std", "risc0-zkp/prove"]
```

Comment:
I'm not sure we need to feature flag the `Program`

Reply:
In 14fb44d847c432302566aef44d18bf5b1e1fef51 I refactored to drop that feature flag and move the control ID calculation into `risc0-zkvm`. Recursion doesn't have anything comparable to `Loader` for `rv32im`, so I simply placed it in the module as a function.

Reply:
Does the WIP bootstrap need it? I thought it called it to compute the ControlIDs.

Reply:
Ok, I actually realized that doesn't work out. I've reverted that commit and instead just hid the function behind a `prove` flag. Not a huge fan of introducing a `prove` flag to `binfmt`, but its the best option I could think of.